common/src/main/java/org/jboss/pnc/common/util/ReadEnvProperty.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2018 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.common.util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ReadEnvProperty {
```

Comment:
I see that we read env properties in PullingMonitor, but IMHO it is wrong as it should use the config file as the rest of the application. Therefore I'd go with this deviation further and do not create a new class for it. We should refactor PullingMonitor instead and use config file approach and set default values in the config entity.

Reply:
I wonder what was the reason to not use the default configs in the PullingMonitor.

Reply:
It was just so as to make it optional. And not change configs. But if you want it in pnc-config.json, we can do this.